packages/language-service/ivy/language_service.ts

```diff
@@ -172,6 +174,31 @@ export class LanguageService {
     return result;
   }
 
+  getCompilerOptionsDiagnostics(): ts.Diagnostic[] {
+    const project = this.project;
+    if (!(project instanceof ts.server.ConfiguredProject)) {
+      return [];
+    }
+
+    const diagnostics: ts.Diagnostic[] = [];
+    const configSourceFile = ts.readJsonConfigFile(
+        project.getConfigFilePath(), (path: string) => project.readFile(path));
+
+    if (!this.options.strictTemplates && !this.options.fullTemplateTypeCheck) {
+      diagnostics.push({
+        messageText:
+            'Consider enabling strict mode. Some features are not available without the strict flag.',
```

Comment:
```suggestion
            'Consider enabling strict mode in the angularCompilerOptions. Some features are not available without the strict flag.',
```
It might be useful to say exactly what/where "strict mode" is if this messageText is going to be displayed in its current form. The only thing that indicates it's related to the compiler is the function which returns it.